### DISCUSSION TITLE
No documentation for loadTranslations, addTranslations, setRoute, setLocale

### DISCUSSION THREAD

**Discussion by galaczi** [sveltekit-i18n/lib#164] [General]
I couldn't find any documentation for loadTranslations, addTranslations, setRoute, setLocale. As a Svelte newbie I am having a hard time understanding the advanced route example.

**Comment by jarda-svoboda** [comment]
Hi @galaczi.
It's documented in the [@sveltekit-i18n/base](https://github.com/sveltekit-i18n/base) docs:
[loadTranslations](https://github.com/sveltekit-i18n/base/tree/master/docs#loadtranslations-locale-string-route-string--promisevoid)
[addTranslations](https://github.com/sveltekit-i18n/base/tree/master/docs#addtranslations-translations--locale-string-recordstring-any--keys-recordstring-string--undefined--void)
[setRoute](https://github.com/sveltekit-i18n/base/tree/master/docs#setroute-route-string--promise)
[setLocale](https://github.com/sveltekit-i18n/base/tree/master/docs#setlocale-locale-string--promise)
